ext/ext.md

```diff
@@ -0,0 +1,56 @@
+# _ext - Index of Distribution Extensions
+
+## Summary
+
+This base extension is used to discover and return an array of extensions.
+
+## Reference Explanation
+
+The base extension may be queried through a `GET` to discover available
+extensions.
+
+Registry level extensions may be discovered with a standard GET as follows.
+
+```HTTP
+    GET /v2/_ext/
```

Comment:
```/v2/_oci/ext``` , is oci considered as the default namespace for all registry operations?

Reply:
I see OCI as the goal for things added that aren't vendor specific. They should be used by extensions that have been standardized/approved by OCI. If a specific registry implementation adds their own extensions, they should be doing so under their own namespace.

Reply:
I think if we follow these conventions, it would be something like `/v2/_ext/discover`, since the spec sits on the `_ext` namespace to surface extensions. This would also allow the spec to add other extension operations in the future.

Reply:
For the vendor-specific question:
All registries build extensions, to fill the gaps in the specs. Since the cloud-hosted registries run on their domains, apis outside of `/v2/` hasn't been a problem.
What came up with the ORAS Referrers API is how this would get implemented by harbor, cncf distribution, jfrog, where they can run their registries in a users k8s cluster. Having APIs be out of the `/v2` path provides routing challenges. 

The premise around the extensions is to enable optional OCI capabilities, but also enable registry-specific capabilities to have a well-defined location to run within. If harbor, gcr, acr, ecr, jrog, zot are implemented on these paths, they just need names that keep them unique. For instance, Alibaba Container Registry is also acr :), so azure/acr might be a path acr might use.
The point is I think we do need a well-defined way for optional extensions, whether they're vendor-specific extensions or community extensions like a scan result indexing tool that @hectorj2f mentions. k8s has an extension model, how can we apply one to the distribution spec?